src/app/Library/CrudPanel/Traits/HasViewNamespaces.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Backpack\CRUD\app\Library\CrudPanel\Traits;
+
+trait HasViewNamespaces
+{
+    private $viewNamespaces = [];
+
+    /**
+     * Return the namespaces stored for the given domain.
+     *
+     * @param  string  $domain  (eg. fields, filters, buttons)
+     * @return array
+     */
+    private function getViewNamespacesFor(string $domain)
+    {
+        return $this->viewNamespaces[$domain] ?? [];
+    }
```

Comment:
I don't think  we need this `private` function any more. The all one is enough, right?

```suggestion

    /**
     * Return the namespaces stored for the given domain.
     *
     * @param  string  $domain  (eg. fields, filters, buttons)
     * @return array
     */
    private function getViewNamespacesFor(string $domain)
    {
        return $this->viewNamespaces[$domain] ?? [];
    }
```